ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -381,6 +381,8 @@ SFVSC:
   - supply_fan_speed_percentage_sensor
   - supply_fan_current_sensor
   - supply_fan_power_sensor
+  - supply_fan_run_mode
```

Comment:
not sure i would want this field on this type, given run_mode states are {AUTO, MANUAL} and probably are more related to the running of the fan than the VFD. please explain what you intend to do with this data.

Reply:
Updated in SFSS